src/api/schema.js

```diff
@@ -50,6 +50,7 @@ import {
   schema as cannedResponseSchema,
   resolvers as cannedResponseResolvers
 } from "./canned-response";
+import { schema as tagSchema, resolvers as tagResolvers } from "./tag";
```

Comment:
Nit: remove unused imports.
```suggestion
import { schema as tagSchema } from "./tag";
```

Reply:
removed that